iam/api-client/src/test/java/iam/snippets/ServiceAccountTests.java

```diff
@@ -85,10 +86,12 @@ public void stage1_testServiceAccountsList() {
   }
 
   @Test
-  public void stage2_testServiceAccountRename() {
-    RenameServiceAccount.renameServiceAccount(PROJECT_ID, SERVICE_ACCOUNT);
+  public void stage2_testServiceAccountRename() throws IOException {
+    String renameTo = "your-new-display-name";
+    RenameServiceAccount.renameServiceAccount(PROJECT_ID, SERVICE_ACCOUNT, renameTo);
     String got = bout.toString();
     assertThat(got, containsString("Updated display name"));
+    assertThat(got, containsString(renameTo));
```

Comment:
Use the returned value from `RenameServiceAccount.renameServiceAccount()` to assert.